app/src/features/rules/screens/rulesList/components/RulesList/RulesList.tsx

```diff
@@ -74,6 +74,11 @@ const RulesList: React.FC<Props> = () => {
           <>
             <div className="rq-rules-list-container">
               {isFeatureLimiterOn && user.isLimitReached ? <FeatureLimiterBanner /> : null}
+
+              {/* TODO: Temp Breadcrumb */}
+              <div className="rq-rules-table-breadcrumb">
+                <span className="breadcrumb-1">Rules</span> {" > "} <span className="breadcrumb-2">All</span>
```

Comment:
Let's hide this till its not completed

Reply:
Looks weird without this. So keeping this for now